### PR TITLE
HS-1142757 - Organizations links on Settings aren't rendering correctly

### PR DESCRIPTION
**Pull request by dr-bizz**
## Description
[HelpScout issue](https://secure.helpscout.net/conversation/2578696149/1142757?folderId=7296147)
The Organization links are only shown if the user is an admin. However, on the old MPDx, users who weren't admins but managed at least one organization could see the page.

This PR allows users who manage an organization and aren't an admin to view the links and visit the page.

## Checklist:

- [ ] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [ ] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [ ] I have requested a review from another person on the project

### CODE REVIEW

File: src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx [hunk 1]
```diff
@@ -54,6 +60,31 @@ const FilterList = styled(List)(({ theme }) => ({
   },
 }));
 
+type ShowMenuItemProps = {
+  item: NavItems;
+  user: Session['user'];
+  hasOrganizationsAccess: boolean;
+};
+
+const showMenuItem = ({
```
Comment (canac) [inline src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx:69]: I like that you pulled this out into a separate function!

File: src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx [hunk 2]
```diff
@@ -54,6 +60,31 @@ const FilterList = styled(List)(({ theme }) => ({
...
+  item,
+  user,
+  hasOrganizationsAccess,
+}: ShowMenuItemProps): boolean => {
+  if (item?.grantedAccess?.length) {
+    if (hasOrganizationsAccess && ['organizations'].includes(item.id)) {
```
Comment (canac) [inline src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx:75]: Is there a reason we're not doing this?

```suggestion
    if (hasOrganizationsAccess && item.id === 'organizations')) {
```

File: src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.graphql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+query ManageOrganizationsAccess {
+  user {
+    administrativeOrganizations(first: 2) {
```
Comment (canac) [inline src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.graphql:3]: We only check that there is at least one org, so we only need to one, right?

```suggestion
    administrativeOrganizations(first: 1) {
```

**Review by canac — APPROVED** [review]
This looks great! I don't think that staging built successfully because it isn't working in staging, but I impersonated the HelpScout user, and it works great locally. I just left a couple of minor questions/suggestions.

File: src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx (hunk 2 above)
Comment (dr-bizz) [reply src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx:75]: Yes, as there are 3 organization links `organizations`, `organizations/accountLists` & `organizations/contacts'`

File: src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.graphql (hunk 3 above)
Comment (dr-bizz) [reply src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.graphql:3]: That is correct, I can change it to 1

File: src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx (hunk 2 above)
Comment (canac) [reply src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx:75]: In that case, don't you want `item.id.includes('organizations')` or `item.id.startsWith('organizations')`? What you have right how will only match 'organizations'.

Comment (dr-bizz) [reply src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx:75]: You're right, it's weird that the links were showing

**Comment by dr-bizz** [comment]
@canac I've pushed it to staging

**Comment by canac** [comment]
@dr-bizz Works great in staging. Thanks for fixing this! 🎉

**Pull request merged**